examples/peripherals/gpio/generic_gpio/main/gpio_example_main.c

```diff
@@ -38,9 +38,21 @@
 #define GPIO_OUTPUT_IO_0    CONFIG_GPIO_OUTPUT_0
 #define GPIO_OUTPUT_IO_1    CONFIG_GPIO_OUTPUT_1
 #define GPIO_OUTPUT_PIN_SEL  ((1ULL<<GPIO_OUTPUT_IO_0) | (1ULL<<GPIO_OUTPUT_IO_1))
+/*
+ * In binary representation,
+ * 1ULL<<GPIO_OUTPUT_IO_0 is equal to 0000000000000000000001000000000000000000 and
+ * 1ULL<<GPIO_OUTPUT_IO_1 is equal to 0000000000000000000010000000000000000000
+ * GPIO_OUTPUT_PIN_SEL                0000000000000000000011000000000000000000
+ * */
```

Comment:
I'm afraid this comment can be confusing if user changed the GPIO number in the menuconfig. Please add an assumption statement to the note, e.g. "Let's say, GPIO_OUTPUT_IO_0=18, GPIO_OUTPUT_IO_1=19".

Reply:
Hi @suda-morris , I agree. Will make the change and see if this woks better.

Thanks

Reply:
Hi @suda-morris , I just made the change. What do you think?👍